main.go

```diff
@@ -109,6 +110,13 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Drill")
 		os.Exit(1)
 	}
+	if err = (&pgbench.Reconciler{
```

Comment:
This part function also be beautified with a loop. We should address this as part of an other PR.